app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -0,0 +1,79 @@
+
+##Study as a postgraduate student
+You can study for [a PGCE or PGDE](/what-is-a-pgce) (an academic qualification) at a university. This is one way to get QTS. Most of the time you will be in schools on placement but there is some academic learning. Courses typically take one year but you may be able to study part-time.
+
+Fees for full-time courses are usually around £9,250 but you can:
+
+- get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects
+- [apply for a student loan](/funding-your-training#tuition-fee-and-maintenance-loans)
+
+Fees are higher for international students.
+
+You should check the fees for a particular course you are interested in with the course provider.
+
+##Train in school (fee paying)
+There are different programmes that focus on training you while you teach in a school:
+
+- School-centred initial teacher training (SCITT)
+- School Direct (fee-funded)
+
+SCITT programmes are delivered by a network of schools, providing school-based training. You won’t be paid a salary while you are training and will pay fees.
+
+School Direct (fee-funded) programmes are delivered by groups of schools in collaboration with a SCITT or university. The content and structure of a School Direct course is broadly similar to a SCITT course and they don't pay you a salary. Some SCITT and School Direct (fee-funded) programmes give you a PGCE or PGDE in addition to QTS. Courses typically take one year but you may be able to study part-time.
+
+Fees are usually around £9,250 but you can:
+
+- get [funding that you do not have to pay back](/funding-your-training#bursaries-and-scholarships) for certain subjects
+- [apply for a student loan](/funding-your-training#tuition-fee-and-maintenance-loans)
+
+You should check the fees for a particular course you are interested in with the course provider.
+
+##Earn a salary while you train
+These courses are usually only suitable for people who already have relevant teaching experience or are already working as unqualified teachers. You may be able to study part-time.
+
+[Read about Nathan’s experience of salaried teacher training](/my-story-into-teaching/teacher-training-stories/salaried-teacher-training-classroom-learning)
+
+###School direct (salaried)
+If you have the right experience, School direct (salaried) give you QTS with many courses also offering a PGCE or PGDE. It takes a year to complete.
+
+[Find out more about School direct](https://www.ucas.com/teaching-option/school-direct-salaried) (salaried)
+
+###Postgraduate teaching apprenticeship
+Apprenticeships offer a year of school-based training which gives you QTS. Some also offer a PGCE or equivalent qualification.
+
+[Find out more about apprenticeships](https://www.gov.uk/guidance/provide-training-for-postgraduate-teaching-apprenticeships)
```

Comment:
This link is for apprenticeship providers - this gov.uk page links back to a page on the GIT legacy site for candidates!

Suggest not having a link here - we need to add some proper apprenticeships content for candidates

Reply:
I've removed this altogether for now. It's not much use without a link and that's what you get when you google it, which is what users will do without a link.